src/components/OcDatepicker.vue

```diff
@@ -1,24 +1,37 @@
 <template>
-  <datetime
-    v-model="update"
-    class="oc-datepicker"
-    :title="title"
-    :type="type"
-    :phrases="phrases"
-    :min-datetime="minDatetime"
-    :max-datetime="maxDatetime"
-    :placeholder="placeholder"
-    @cancel="$_cancel()"
-    @input="$_input()"
-  ></datetime>
+  <div>
+    <label class="oc-label" :for="id" v-text="label" />
```

Comment:
I am not quite sure but we might need an value attribute for accessibility?
![image](https://user-images.githubusercontent.com/26169327/111169513-5197d300-85a3-11eb-971e-82615f657d0e.png)

Reply:
Need to do some research on that 👀 

But I don't think it's required. The value here is basically what the user inputs. The same goes for the `text-input-fields` component, where we also don't have a value attribute. The calendar-modal itself may be another story, though...

Reply:
Research done on my side -> not required